platform/php/MailChecker.php

```diff
@@ -25,9 +30,10 @@ public static function isValid(string $email): bool
         return self::validEmail($email) && !self::isBlacklisted($email);
     }
 
+    /** @return array<string, true> */
```

Comment:
@lyrixx This is wrong, since we're returning array_keys, this is now `array<string>`

Reply:
up for a PR?